Formula/kubetail.rb

```diff
@@ -0,0 +1,14 @@
+class Kubetail < Formula
+  desc "Tail logs from multiple Kubernetes pods at the same time"
+  homepage "https://github.com/johanhaleby/kubetail"
+  url "https://github.com/johanhaleby/kubetail/archive/1.6.8.tar.gz"
+  sha256 "c6c7ffa54d6f5d28b8c3fa6a6a505827f0a0d4887d8376d6fc605b1bf7c8ae64"
+
```

Comment:
You need a dependency on a formula that provides kubectl. The only reason the tests passed were because you didn't really test anything. If your tests were more thorough you would see that kubectl isn't installed by default in Homebrew's sandboxed environment, and the test would have failed.

Reply:
After reading the source, you also clearly depend on `jq` which is not installed by default either. Please add tests that will exercise calls to both `jq` and `kubectl`.